### PR TITLE
Move handle_time to entityset

### LINKED ISSUE #1275: Move _handle_time from Entity to EntitySet
The `Entity._handle_time` method needs to be moved to `EntitySet for` Woodwork integration.

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
### Move `_handle_time` from Entity to EntitySet
Closes #1275 

Moves the `_handle_time` method from Entity to EntitySet in preparation for Woodwork integration.

### CODE REVIEW

File: featuretools/computational_backends/calculate_feature_matrix.py [hunk 1]
```diff
@@ -177,10 +177,11 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
 
         if instance_ids is None:
             index_var = target_entity.index
-            df = target_entity._handle_time(target_entity.df,
-                                            time_last=cutoff_time,
-                                            training_window=training_window,
-                                            include_cutoff_time=include_cutoff_time)
+            df = entityset._handle_time(datatable_id=target_entity.id,
```
Comment (gsheni) [inline featuretools/computational_backends/calculate_feature_matrix.py:180]: Let's keep entity_id for now.

Comment (thehomebrewnerd) [reply featuretools/computational_backends/calculate_feature_matrix.py]: Changed back to `entity_id` for now.

**Review by rwedge — APPROVED** [review]
Looks good

**Comment by wuxinshui** [comment]
unsubscribe



At 2020-12-15 23:14:47, "Nate Parsons" ***@***.***> wrote:

@thehomebrewnerd commented on this pull request.

In featuretools/computational_backends/calculate_feature_matrix.py:

> @@ -177,10 +177,11 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
 
         if instance_ids is None:
             index_var = target_entity.index
-            df = target_entity._handle_time(target_entity.df,
-                                            time_last=cutoff_time,
-                                            training_window=training_window,
-                                            include_cutoff_time=include_cutoff_time)
+            df = entityset._handle_time(datatable_id=target_entity.id,


Changed back to entity_id for now.

**Pull request merged**